posts/2020-03-31-Rust-survey-2019.md

```diff
@@ -0,0 +1,216 @@
+* "Better platform support"
+* "Security and performance, cost efficient and "green" (low carbon footprint) language"
+* "Embedded development targeting ARM"
+* "Better GUI framework, similar to Qt or directly using Qt via bindings."
+
+Most indicated that Rust maturity - such as more libraries and complete learning resources and more mature production capabilities - would make Rust more appealing.
+
+Let's take a closer look at each of these, starting with the need for more mature libraries.
+
+## Libraries - a Closer Look
+
+When we asked users what libraries they consider critical to the Rust ecosystem, these were the top ten responses:
+* serde
+* rand
+* tokio
+* async
+* clap
+* regex
+* log
+* futures
+* hyper
+* lazy_static
+
+We also asked how many dependencies users were using were 1.0 or above.
+* 0.8% indicated "All"
+* 6.7% indicated "Most"
+* 65.9% indicated "Some"
+* 5.2% indicated "None"
+* 21.4% indicated "I don't know"
+
+[Add COPY/analysis here]
+
+## IDEs and Tooling - a Closer Look
+
+IDE support for Rust was also cited as a barrier to adoption.
+
+When we asked users what editors they use, Vim and VSCode were the most popular by far, followed by Intellij.
+
+We also asked what IDE setups users used:
+* 43.3% indicated RLS
+* 21.7% indicated Intellij
+* 15.2% indicated Rust-analyzer
+* 12.4% indicated No (or CTAGS)
+* 4.2% indicated Only Racer
+
+As for platforms that users develop on - Linux and Windows continue to dominate. 
+* 55% of Rust users develop on Linux
+* 24% develop on Windows
+* 23% develop on macOS
+
+![Which Rust version do you use for your applications?](images/2020-03-RustSurvey/16-Which-Rust-Version.png)
+
+The vast majority of users use the current stable version of Rust (63%). Surprisingly, the number of users using the Nightly compiler in their workflow is down at 20%. Last year it was at over 56%.
+
+[ADD COPY/analysis]
+
+## Learning Curve - a Closer Look
+
+Rust is well known for its significant learning curve. 
+
+![How would you rate your Rust expertise](/images/2020-03-RustSurvey/13-Rust-Expertise-and-Time-In-Rust.png)
+
+We asked survey respondents to rate their own Rust expertise.
+
+![How long did it take to be productive](/images/2020-03-RustSurvey/8-How-Long-To-Be-Productive.png)
+
+Next, in order to better understand the learning curve, we asked how long it took users to feel that they were productive with Rust.
+
+About 37% of Rust users felt productive in Rust in less than a month of use - this is not too different from the percentage last year (40%). Over 70% felt productive in their first year. Unfortunately, like last year, there is still a struggle among users - 21% indicated they did not yet feel productive.
+
+![How long have you been unproductive](/images/2020-03-RustSurvey/9-How-Long-Unproductive.png)
+
+Additionally, many users indicated that they were not feeling comfortable in Rust - even after over 3 months of use. This is likely a result of Rust's high learning curve.
```

Comment:
Nit: I would delete the last sentence and merge this with the next paragraph, I think.